src/cmd/sql_cmd.h

```diff
@@ -1263,7 +1265,7 @@ bool HandleLoadDataInfile(const std::string& database, const std::string& table,
         }
         ++i;
     } while (std::getline(file, line));
-    LOG(INFO) << "load " << i << " rows";
+    std::cout << "Load " << i << " rows";
```

Comment:
After `HandleLoadDataInfile`, we will print 'succeed' msg. We want a msg like 'SUCCEED: load xxx rows'
It's ok to print here, but you should delete the 'succeed' msg after `HandleLoadDataInfile`.
Or `HandleLoadDataInfile` returns the rows count.